File: SPAtraductores/SPAtraductores/Models/Peticion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SPAtraductores.Models
{
    public class Peticion
    {
        public int IdIdioma { get; set; }
        public int IdServicio { get; set; }
        public int idTraductor { get; set; }
        public String NombreSolicitante { get; set; }
        public String Descripcion { get; set; }
        public String Email { get; set; }
        public String Tlfn { get; set; }
    }
}

File: SPAtraductores/SPAtraductores/ClientApp/app/Models/Peticion.ts

export class Peticion {

    constructor(
        public idIdioma: number,
        public idServicio: number,
        public idTraductor: number,
        public nombreSolicitante: string,
        public descripcion: string,
        public email: string,
        public tlfn: number
    ) { }
}